store/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -1333,6 +1337,12 @@ func (mvcc *MVCCLevelDB) ResolveLock(startKey, endKey []byte, startTS, commitTS
 	mvcc.mu.Lock()
 	defer mvcc.mu.Unlock()
 
+	if len(startKey) > 0 {
```

Comment:
What's this?

Reply:
Temproraly solving some mvcc leveldb problem  without which resolve lock will not work for splitted regions.